js/client-teambuilder.js

```diff
@@ -1245,6 +1254,39 @@
 			this.save();
 		},
 
+		setNotesBlur: function (e) {
+			var textarea = $(e.currentTarget);
+			if (!$.trim(textarea.val())) textarea.parent().hide();
+			if (textarea.val().indexOf('|') >= 0 || textarea.val().indexOf(']') >= 0 || textarea.val().indexOf('\t') >= 0) {
+				app.addPopupMessage("Unfortunately, set notes can't contain tabs or the characters | and ], since they're used for storing teams.");
```

Comment:
I had been looking at [W3's spec](https://www.w3.org/TR/webstorage/#the-localstorage-attribute), guess I should have looked at the browsers' implementations instead.
>Perhaps we should only support set notes if persistent storage is on?

It seems that only newer chrome and opera versions support StorageManager.persist() so far, so that'd require waiting until it's more widely adopted I suppose.
Also, I'm not sure if I'm reading this correctly, but my impression is that local storage would only get cleared if the machine is running out of hard drive space. Are <200KB of team data really going to be relevant when it comes to that?